package.json

```diff
@@ -7,7 +7,9 @@
     "lint": "yarn workspaces run lint",
     "compile-palette": "yarn workspace @artsy/palette compile",
     "type-check": "yarn compile-palette && yarn workspaces run type-check",
-    "build-docs": "yarn compile-palette && yarn workspace artsy-palette-docs build"
+    "build-docs": "yarn compile-palette && yarn workspace artsy-palette-docs build",
+    "storybook": "yarn workspace @artsy/palette storybook",
```

Comment:
Could be a follow-up, can we update the README / docs with notes on this new change / workflow?